### PR TITLE
Add SQL guide

### PR DESCRIPTION
**Pull request by stevhliu**
This PR adapts @nateraw's awesome SQL notebook as a guide for the docs!

### CODE REVIEW

**Comment by nateraw** [comment]
I think we may want more content on this page that's not SQL related. Some of that content probably already lives in the main `load` docs page, but might be bad to remove major things like csv/pandas from there...WDYT we should do @lhoestq ?

**Review by mariosasko — COMMENTED** [review]
I agree with @nateraw and think we should expand the existing "Load" page with more SQL examples instead of starting a new page unless there is enough content related to tabular data to put there (it seems that's not the case for now). 

PS: CSV should be the preferred way to store/load tabular data IMO, due to its ease of use.

File: docs/source/tabular_load.mdx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+```py
+>>> ds.filter(lambda x: x["cases"] > 10000)
+```
+
+## PostgreSQL
+
+<Tip warning={true}>
+
+This example is designed to only run in a Google Colab. Be careful if you want to run the server commands locally!
```
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/tabular_load.mdx:81]: Thanks for the feedback, put CSV first in this section and replaced the whole PostgreSQL section with a link to the actual notebook.

**Review by lhoestq — APPROVED** [review]
Cool thanks !

**Pull request merged**